managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+
+   ![Service Dialog options](../images/embedworkflow_servicedialogdynamic.png){: caption="Service Dialog options" caption-side="bottom"}
+
+* Service Catalog Items
+
+   The provisioning, configuring, or retiring entry points now include the option for using **Embedded Workflows** or **Embedded Automate**.
+
+   ![Entry Points options](../images/embedworkflow_entrypointsembedded.png){: caption="Entry Points options" caption-side="bottom"}
+
+### Adding an Embedded Workflows Repository
+
+As a starting point to begin authoring a workflow, you must first add the repository that includes your workflows.
+
+To add a repository, complete the following steps:
+
+1. Click  **Automation** > **Embedded Workflows** > **Repositories** > **Configuration** > **Add New Repository**.
+
+   ![Add a repository](../images/embedworkflow_addnewrepository.png){: caption="Add a repository" caption-side="bottom"}
+
+2. Enter *Name*, *Description*, and a *GitHub URL*.
+
+3. Provide a Repository Name in the **Name** field.
+
+4. Add a description for the repository in the **Description** field.
+
+5. Add a **GitHub URL** for the repository.
+
+6. Select the appropriate SCM (Source Control) credentials from the **SCM Credentials** dropdown menu. These are the credentials for how you access GitHub, and are used in different ways within embedded workflows.
+
+7. Provide a branch name in the **SCM Branch** field for the branch that you want to pull.
+
+8. Click **Save**. Upon saving, the repository is imported and scanned for any included workflows.
+
+   ![Repository details](../images/embedworkflow_addnewrepository_details.png){: caption="Repository details" caption-side="bottom"}
+
+9. Confirm that the synchronization with your repository is successful by viewing the list of workflows that are detected for the repository.
+
+**Note:** If you are adding a repository that is not publicly available, add an SCM (source control) credential for accessing the repository. This credential is separate from the workflow credentials that you configure in the procedure for adding the repository. Source Control credentials have several attributes that can be configured:
+
+   - **Username**: The username to use in conjunction with the source control system.
+   - **Password**: The password to use in conjunction with the source control system.
+   - **Private key passphrase**: If the SSH private key used is protected by a passphrase, you can configure a key passphrase for the private key.
+   - **Private Key**: Copy or drag and drop the actual SSH Private Key to be used to authenticate the user to the source control system by using SSH.
+
+### Viewing the List of Workflows in a Repository
+
+When your repository is added, the repository shows in the *Repositories* list. You can now click the entry for the repository to open it so that you can scan for existing workflows.
```

Comment:
> so that you can scan for existing workflows.

This sounds like clicking on the repository is initiating the scan (maybe just overloaded terms).  How about "so that you can view the imported workflows"

Reply:
I have updated this sentence to be: You can now click the entry for the repository to open it so that you can view the imported workflows.